relayer/src/__tests__/target.test.ts

```diff
@@ -0,0 +1,46 @@
+import Target from "../target";
+import {
+    apiMock, loggerMock, txDataMock
+} from "../mocks";
+// import { ApiPromise } from "@polkadot/api";
+import { of } from "rxjs";
+
+describe("Target class", () => {
+    const defaultParams = {
+        api: apiMock,
+        signer: "random signer address",
+        logger: loggerMock,
+    };
+
+    const blockSubscriptions = of(txDataMock);
+
+    it("should create an instance", () => {
+        const target = new Target(defaultParams);
+
+        expect(target).toBeInstanceOf(Target);
+        expect(target).toHaveProperty("processSubscriptions");
```

Comment:
Same comment about method vs property

Reply:
I agree, it's not very useful - these are just basic tests, for now, I will dedicate more time to implement proper ones once we have a demo-able version